CONTRIBUTING.md

```diff
@@ -22,6 +22,8 @@ First time contributing to Homebrew? Read our [Code of Conduct](https://github.c
 * `git commit` with message formatted `foo 2.3.4 (new formula)`
 * [open a pull request](https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request) and fix any failing tests
 
+Once you've addressed any potential feedback and a member of the Homebrew org has approved your pull request, the [BrewTestBot](https://github.com/BrewTestBot) will automatically merge it a couple of minutes later.
```

Comment:
It's not exclusively testbot by the way, it can still be manually merged by maintainers.

Reply:
What about this wording?
```suggestion
Once you've addressed any potential feedback and a member of the Homebrew org has approved your pull request, your PR will be automatically merged by our [@BrewTestBot](https://github.com/BrewTestBot) a couple of minutes later – if not merged by a maintainer manually before.
```

Reply:
I think I'd prefer 

> Once you've addressed any potential feedback and a member of the Homebrew org has approved your pull request, your PR will be automatically merged by our [@BrewTestBot](https://github.com/BrewTestBot), or a maintainer.